src/headingengine.js

```diff
@@ -30,6 +23,39 @@ export default class HeadingEngine extends Plugin {
 	/**
 	 * @inheritDoc
 	 */
+	constructor( editor ) {
+		super( editor );
+
+		const t = editor.t;
+
+		/**
+		 * A set of default localized labels for `config.heading.formats`.
+		 *
+		 * @readonly
+		 * @protected
+		 * @member {Object} #_localizedFormatLabels
+		 */
+		const labels = this._localizedFormatLabels = {
```

Comment:
Because of the above https://github.com/ckeditor/ckeditor5-heading/pull/57#discussion_r103485274. Both default config and user config must be localized. The first is localized in `constructor()`, the second in `init()`. I cannot share these labels in any other way because they require `editor.t()` to be created.